packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -519,18 +523,18 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
         const {seedingChannelId, leechingChannelId} = wire.paidStreamingExtension;
         if (leechingChannelId) {
           log.info(`About to close payment channel ${leechingChannelId}`);
-          await this.paymentChannelClient.getLatestPaymentReceipt(leechingChannelId);
-          await this.paymentChannelClient.closeChannel(leechingChannelId);
-          wire.paidStreamingExtension.leechingChannelId = null;
-          log.info(`Channel ${leechingChannelId} closed`);
+          return this.paymentChannelClient.closeChannel(leechingChannelId).then(() => {
```

Comment:
I think @andrewgordstewart already asked this, but is there a reason to use `.then` promise syntax here instead of await?

Reply:
I wanted to chain promises, but I'll make an extra function and use that instead, so that I maintain consistency.

Reply:
Do we need a separate function? Would this work?
```suggestion
          await this.paymentChannelClient.closeChannel(leechingChannelId);
          wire.paidStreamingExtension.leechingChannelId = null;
          log.info(`Channel closed: ${leechingChannelId}`);```

Reply:
https://github.com/statechannels/monorepo/pull/1969/commits/034e40f790457b167989f3792ebeda37eaecdbb4

Reply:
> Do we need a separate function? Would this work?

this would close the channel one by one, unless i'm wrong.